### PR TITLE
[DM-24733] Added support of network policy

### PR DESCRIPTION
**Pull request by shivanshs9**
### Network Policies
- [x] Added default-deny policy
- [x] All connections to Qserv czar proxy should then be opened
- [x] Replication controller should be able to connect to czar database
- [x] Replication worker should be able to connect to replication database
- [ ] Connection between replication controller and replication worker should be clarified
- [ ] Connection between xrootd components should be clarified
   - [x] Redirector CMSD port <- Workers

### Integration Tests:
- [x] Passing
    - Failing since all ingress traffic is blocked.

### CODE REVIEW

**Comment by shivanshs9** [comment]
@fjammes Travis build doesn't include Kind with network plugin enabled. Is it possibly the reason for failure?

File: .travis.yml [hunk 1]
```diff
@@ -3,24 +3,24 @@ language: go
...
 go:
-- 1.13.4
+  - 1.13.4
 
 before_script:
   - if [ "$DOCKER_USERNAME" ]; then docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD"; fi
   - ./install-operator-sdk.sh
-  - git clone --depth 1 -b "v0.6.0" --single-branch https://github.com/k8s-school/kind-travis-ci.git
-  - ./kind-travis-ci/kind/k8s-create.sh -s
+  - git clone --depth 1 -b "master" --single-branch https://github.com/k8s-school/kind-travis-ci.git
```
Comment (fjammes) [inline .travis.yml]: Maybe using a tag here would be more robusts, please use `git tag` on kind-travis-ci repository to pick latest one.

File: base/qserv.yaml [hunk 2]
```diff
@@ -5,6 +5,7 @@ metadata:
 spec:
   storageclass: "standard"
   storagecapacity: "1Gi"
+  securenetwork: true
```
Comment (fjammes) [inline base/qserv.yaml]: Maybe rename it to  "networkpolicies"

File: .travis.yml (hunk 1 above)
Comment (shivanshs9) [reply .travis.yml]: I'll use the [`k8s-v1.18.2`](https://github.com/k8s-school/kind-travis-ci/commits/k8s-v1.18.2) tag then?

File: base/qserv.yaml (hunk 2 above)
Comment (shivanshs9) [reply base/qserv.yaml]: Yeah sure!

**Pull request merged**